SwitchDemo/SwitchDemo/Activity1.cs

```diff
@@ -0,0 +1,23 @@
+namespace SwitchDemo
+{
+    [Activity(Label = "SwitchDemo", MainLauncher = true)]
+    public class Activity1 : Activity
+    {
+        protected override void OnCreate(Bundle? bundle)
+        {
+            base.OnCreate(bundle);
+
+            SetContentView(Resource.Layout.Main);   
+
+            var s = FindViewById<Switch>(Resource.Id.monitored_switch);
+
+            ArgumentNullException.ThrowIfNull(s);
+            s.CheckedChange += delegate (object? sender, CompoundButton.CheckedChangeEventArgs e) {
+
+                var toast = Toast.MakeText(this, "Your answer is " + (e.IsChecked ? "correct" : "incorrect"),
+                                            ToastLength.Short);
+                toast?.Show();
```

Comment:
This one is weird... This doesn't look like it can return null, but Google didn't decorate it with `@NotNull`:

https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/widget/Toast.java;l=492

What you did here is great! I'm just thinking if there is something we should do to make this better in xamarin/xamarin-android.